week7/src/components/Input.js

```diff
@@ -0,0 +1,17 @@
+import imgSrc from "../assets/Search.png";
+import "../components/Header.css";
+import { useMediaQuery } from "react-responsive";
+function Input() {
+  const isTablet = useMediaQuery({ maxWidth: 1199 });
+  return (
+    <div className={isTablet ? "search-tablet-box" : "search-box"}>
+      <img className="search-img" src={imgSrc} alt="검색" />
+      <input
+        className={isTablet ? "search-tablet" : "search"}
+        placeholder="링크를 검색해 보세요."
+      ></input>
```

Comment:
input이 검색은 안되지만 그래도 `useState`를 이용해서 `value`와 `onChange` 핸들러까지는 달아주면 좋을 것 같습니다.

Reply:
그리고 추가적으로 form 태그로 input을 감싸는게 좀 더 시멘틱하고 나중에 많은 처리를 하기에 도움이 됩니다.

```suggestion
 <form>
      <input
        className={isTablet ? "search-tablet" : "search"}
        placeholder="링크를 검색해 보세요."
      ></input>
      
      </form>
```